### PR TITLE
Feat: Added Snackbar

### PR DESCRIPTION
**Pull request by anamkaa**
Added variations of snackbar
Screenshot (258)

### CODE REVIEW

File: stylesheets/components/snackbar.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  font-size: 0.9rem;
+}
+
+.snackbar-wrapper-2 {
+  display: flex;
+  align-items: center;
+  margin-left: auto;
+}
+
+.snackbar-stacked-wrapper {
```
Comment (krishanu2911) [inline stylesheets/components/snackbar.css:45]: just suggestion not mandatory but you can name it as snackbar-stack-wrap like main aim is to shorten the class name it's up to you wanted it or not just suggested
This is also fine no problem with it.

**Review by krishanu2911 — APPROVED** [review]
just added few suggestions (optional) 
Code is good to go and clean well written

**Review by vivekmehto — APPROVED** [review]
Looks good.

File: stylesheets/components/snackbar.css (hunk 1 above)
Comment (anamkaa) [reply stylesheets/components/snackbar.css:45]: Will keep that in mind.

**Pull request merged**